extensions/sql-bindings/package.nls.json

```diff
@@ -1,4 +1,4 @@
 {
-	"%displayName%": "SQL Bindings",
-	"%description%": "Enables users to develop and publish Azure Functions with Azure SQL bindings"
+	"displayName": "SQL Bindings",
+	"description": "Enables users to develop and publish Azure Functions with Azure SQL bindings"
```

Comment:
`Azure SQL bindings` what is this?

Reply:
This extension is specifically SQL bindings-related code that enables the user experience of Azure SQL Bindings. Azure SQL Bindings docs are [here](https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-azure-sql).